.storybook/stories/Button/ExampleButton.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
we will keep these examples?

Reply:
it's a whole new story format going forward. thought an example would be worthwhile for now